yash-datalake/scripts/raw_to_transformed.py

```diff
@@ -0,0 +1,66 @@
+import sys
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import lit
+import datetime
+
+now = datetime.datetime.now()
+year = str(now.year)
+month = str(now.month)
+day = str(now.day)
+today = str(now.date())
+
+
+def main(storage_account_name, container_name):
+    spark = SparkSession.builder.appName("rawToTransformed").getOrCreate()
+
+    '''
+        ********************************************************************************************************
+        '''
+    customer_df = spark.read.csv("wasbs://"+container_name+"@"+storage_account_name+".blob.core.windows.net/datasets/customers")

```

Comment:
uri must be passed in using reference() - here and all the other instances below